docs/DeveloperGuide.md

```diff
@@ -181,7 +181,51 @@ Step5. Execution of this method will result in a call to `MainWindow#selectProje
     * Cons:
         * `MainWindow` and `CommandResult` are not closed to modification. A new instruction to change the UI might require the addition of fields to `CommandResult` (boolean fields for instructions and other fields for related data) as well as a new conditional statement in `MainWindow#execute` to handle the new instruction. This makes it relatively difficult to add new instructions.
 
-### Update Commands
```

Comment:
Please help to change Rc ones too

Reply:
It is already changed! It's at [line 228](https://github.com/AY2021S2-CS2103T-T11-2/tp/pull/364/files#diff-1a95edf069a4136e9cb71bee758b0dc86996f6051f0d438ec2c424557de7160bR228)! :blush:

Reply:
But probably can make `Feature` singular?

Reply:
> But probably can make `Feature` singular?

But I can do this in a new PR. Can merge this I think.

Reply:
@lirc572 okay!